packages/workspace-node/src/workspace/codebase/file/parse/index.ts

```diff
@@ -1,12 +1,22 @@
+import type { ParserOptions } from '@babel/parser';
 import { parse as babelParser } from '@babel/parser';
 
-import { RandomObject } from '../../../../types';
-
-// TODO: check type
-const babelConfig: RandomObject = {
+const babelConfig: ParserOptions = {
   sourceType: 'module',
   createParenthesizedExpressions: true,
-  plugins: ['jsx', ['typescript', { isTSX: true }], 'babel-plugin-recast'],
```

Comment:
I was getting some ts errors so I tried removing this, and so far so good.

Reply:
yeah this is very tricky